### PR TITLE
Migrate to App router

### PR DESCRIPTION
**Pull request by SonMooSans**

### CODE REVIEW

**Comment by Gary50613** [comment]
> @Gary50613 I don't have access to the preview, can you add me to Vercel team?

I want but it requires vercel Pro per member 😔

**Comment by SonMooSans** [comment]
@Gary50613 go review it, only you can do that(x

**Comment by Gary50613** [comment]
> @Gary50613 go review it, only you can do that(x

oh i can create a shard link for u

File: content/docs/commands/chat.mdx [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 description: 跨群聊天讓你不用再額外的加入別的伺服器，就能和別的伺服器的人聊天。透過跨群聊天能夠不用另外加入新的群組，就能和其他群的成員交流。
 ---
 
-# 跨群聊天
-
-import { Adsense, Tip, Warning } from "@components/mdx";
-
 <Tip>
   我們剛剛恢復了跨群聊天的運作！更多的資訊請到我們
   [支援群組](https://discord.gg/yeecord) 關注最新消息
```
[the remark this answers is not in the document]
Comment (SonMooSans) [reply content/docs/commands/chat.mdx:8]: Then open another PR to solve, it's not a problem related to this PR

File: content/blog/discord-bot-troubleshooting.mdx [hunk 2]
```diff
@@ -1,16 +1,13 @@
 ---
-slug: discord-bot-troubleshooting
 title: Discord 音樂機器人沒聲音、錯誤、無法邀請？「5方法」排解機器人問題！
-authors: kane
+authors: [kane]
 description: "你有沒有遇過想要使用 Discord 音樂機器人播放 YouTube 上的音樂時，發現機器人出現錯誤、沒有反應、進去了語音頻道卻沒聲音、按了按鈕之後出現一行紅字？這篇文章將教你五個方法快速的排解這類問題，並且如何跟開發者來回報。"
-tags: ["Discord", "dc", "音樂", "機器人", "錯誤", "沒聲音", "沒反應", "2022"]
+tags: [Discord, "dc", "音樂", "機器人", "錯誤", "沒聲音", "沒反應", "2022"]
```
[the remark this answers is not in the document]
Comment (SonMooSans) [reply content/blog/discord-bot-troubleshooting.mdx]: Seems like Prettier did this

**Comment by Gary50613** [comment]
截圖 2023-08-19 下午7 35 48
is it possible to make default open behavior?

**Comment by SonMooSans** [comment]
> is it possible to make is default open like the old one?
> 
> 截圖 2023-08-19 下午7 35 48

Supported in the latest version of Next Docs

**Review by Gary50613 — APPROVED** [review]
LGTM

**Pull request merged**